extensions/amp-playbuzz/0.1/test/test-amp-playbuzz.js

```diff
@@ -104,30 +100,30 @@ describes.realWin(
 
     function testIframe(iframe, itemSrcUrl) {
       expect(iframe).to.not.be.null;
-      expect(startsWith(iframe.src, itemSrcUrl)).to.be.true;
+      expect(iframe.src.includes(itemSrcUrl)).to.be.true;
```

Comment:
```suggestion
      expect(iframe.src.startsWith(itemSrcUrl)).to.be.true;
```

Reply:
Following the changes I made to the extension, the item src url can be formatted as either: